src/schema/sources.ts

```diff
@@ -367,6 +367,66 @@ export const typeDefs = /* GraphQL */ `
       categoryId: String
     ): SourceConnection!
 
+    """
```

Comment:
That's for debugging right?

Reply:
Yeah just a duplication but using the slave specifically :)